src/module.js

```diff
@@ -780,7 +794,15 @@ export default class DailyIframe extends EventEmitter {
     this._nativeInCallAudioMode = inCallAudioMode;
 
     // If we're in a call now, apply the new audio mode
-    if (this.shouldDeviceUseInCallAudioMode(this._meetingState)) {
+    // (assuming automatic audio device management isn't disabled)
+    const deviceManagementDisabled =
+      this.properties.reactNativeConfig &&
```

Comment:
can we add a convenience method `deviceManagementDisabled('audio|video')` to dry up this logic?

Reply:
yes. i went back and forth on whether to do this while i was writing it and decided not to since it's such a standard pattern (`a && a.b && a.b.c`). that said, reading it over now it does look wordy and gross